Kernel/Process.cpp

```diff
@@ -430,6 +430,17 @@ void signal_trampoline_dummy()
         :
         : "i"(Syscall::SC_sigreturn),
         "i"(offset_to_first_register_slot));
+#elif ARCH(AARCH64)
+    asm(
+        ".global asm_signal_trampoline\n"
+        "asm_signal_trampoline:\n"
+        // TODO: Implement this when we support userspace for aarch64
```

Comment:
:)

Reply:
I feel like I'm quite close actually now that I'm working on handling page faults :)